src/3.1/docker-entrypoint.sh

```diff
@@ -42,6 +42,9 @@ if [ "$1" == "neo4j" ]; then
     setting "dbms.connector.https.listen_address" "0.0.0.0:7473"
     setting "dbms.connector.bolt.listen_address" "0.0.0.0:7687"
     setting "dbms.mode" "${NEO4J_dbms_mode:-}"
+    if [ -n "$NEO4J_dbms_advertisedAddress" ]; then
```

Comment:
We've started using a convention for future improvements, so this variable should be `NEO4J_dbms_connectors_defaultAdvertisedAddress`.

Also, by using a default value as on the next line, you should be able to get rid of the if statement like so:

`setting "dbms.connectors.default_advertised_address" "${NEO4J_dbms_connectors_defaultAdvertisedAddress:-}"`

After you make those changes and test it, I'll be happy to merge it quickly.

Thanks for your contribution!